### PR TITLE
README: Simple Developer Documentation

### PR DESCRIPTION
**Pull request by mrmundt**
To enable easier usage and changes to this site and its subsites, I added a simple README. It describes the purpose and structure of the site, as well as linking to official documentation on how to build and view locally.

**NOTE**: Intended audience of this README is the website maintainers, _NOT_ website users. That is, the README does not show up on the website. It is entirely and purely for maintenance purposes.

### CODE REVIEW

**Comment by mrmundt** [comment]
@charmoniumQ - I added you as a reviewer for one important purpose: Would this README help you, as a new person to the project, understand better how you might try to make changes?

**Review by elaineraybourn — APPROVED** [review]
lines 4 and 8 refer to the reader as the user. In the practice guides, I used "you". What is the best? @rmmilewi and @mrmundt any ideas?

lines 5-6 I would rephrase to read "...recent posts about Productivity and Sustainability Improvement Planning (PSIP)."

line 16 says "this site .... into three sites": consider rephrasing so you don't have a repeated word

nice acknowledgement statement and sections about structure, testing

**Comment by elaineraybourn** [comment]
@mrmundt I submitted my review and approval for changes.

**Comment by mrmundt** [comment]
> lines 4 and 8 refer to the reader as the user. In the practice guides, I used "you". What is the best? @rmmilewi and @mrmundt any ideas?

This README is intended for the PSIP website maintainers, not PSIP website users. That is, the README does not show up on the website. It's just in this repository and is meant to be useful for us (you, me, Reed, Sam, etc.) and anyone who might take over maintenance of the website in the future.

> 
> lines 5-6 I would rephrase to read "...recent posts about Productivity and Sustainability Improvement Planning (PSIP)."
> 

Changed.

> line 16 says "this site .... into three sites": consider rephrasing so you don't have a repeated word
> 

Changed.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+and procedures to enable use of the PSIP process for their project.
+
+This application is built using GitHub Pages on top of the suggested Jekyll
+theme.
+
+## Structure
+
+The website is currently divided into three sections:
+
+- _Main Site_: This is the main site/home page. The source code can be found at https://github.com/bssw-psip/bssw-psip.github.io
```
Comment (charmoniumQ) [inline README.md]:
```suggestion
- _Main Site_: This is the main site/home page, <https://bssw-psip.github.io/>. The source code can be found at <https://github.com/bssw-psip/bssw-psip.github.io>.
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+- _Practice Guides_: This subsite provides the guides for utilizing PSIP to improve team practices. The source code can be found at https://github.com/bssw-psip/practice-guides
```
Comment (charmoniumQ) [inline README.md]:
```suggestion
- _Practice Guides_: This subsite, <https://bssw-psip.github.io/practice-guides/>, provides the guides for utilizing PSIP to improve team practices. The source code can be found at <https://github.com/bssw-psip/practice-guides>.
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+- _Project Tracking Card Catalog_: This subsite provides previously developed and example PTCs, for use in tracking improvement progress. The source code can be found at https://github.com/bssw-psip/ptc-catalog
```
Comment (charmoniumQ) [inline README.md]:
```suggestion
- _Project Tracking Card Catalog_: This subsite, <https://bssw-psip.github.io/ptc-catalog/catalog/>, provides previously developed and example PTCs, for use in tracking improvement progress. The source code can be found at <https://github.com/bssw-psip/ptc-catalog>.
```

**Review by charmoniumQ — APPROVED** [review]
The changes I suggested are quite optional. Lgtm

**Review by elaineraybourn — COMMENTED** [review]
Thanks for making the suggested changes to the text.
